libraries/AP_Proximity/AP_Proximity_MAV.cpp

```diff
@@ -88,26 +88,36 @@ void AP_Proximity_MAV::handle_msg(mavlink_message_t *msg)
         mavlink_msg_obstacle_distance_decode(msg, &packet);
 
         // check increment (message's sector width)
-        uint8_t increment = packet.increment;
-        if (packet.increment == 0) {
+        float increment;
+        if (packet.increment_f > 0) {
```

Comment:
```suggestion
        if (is_positive(packet.increment_f)) {
```

Reply:
would rather leave it > 0 for readability